Examples_Nuget/Docker/Dockerfile.Metadata/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/sdk:5.0
+
+RUN cd /home && \
+  git clone https://github.com/TileDB-Inc/TileDB-Cloud-CSharp.git -b bd/ch13349-metadata-example && \
+  dotnet publish TileDB-Cloud-CSharp/Examples_Nuget/TileDB.ExampleMetadata/metadata.csproj -c Release -o /app/example_metadata
+
+
+RUN cd /app/example_metadata 
+
+WORKDIR /app/example_metadata
```

Comment:
Does it work correctly if you are in a different directory?

Reply:
Yes, if WORKDIR is in different direcotry. it still works